### ISSUE TITLE
Preset-attributify does not work with scoped-svelte mode

### ISSUE THREAD

**Issue by aadi58002** [unocss/unocss#2078]
vite Config
 ```typescript
 import { sveltekit } from '@sveltejs/kit/vite';
import { extractorSvelte } from '@unocss/core';
import UnoCSS from 'unocss/vite';
import {
  presetTypography,
  presetIcons,
  presetUno,
  presetAttributify,
  transformerDirectives,
  transformerVariantGroup,
  transformerCompileClass
} from 'unocss';

const config = {
  plugins: [
    UnoCSS({
      extractors: [extractorSvelte],
      mode: 'svelte-scoped',
      presets: [presetAttributify(), presetTypography(), presetIcons({ scale: 2.0 }), presetUno()],
      transformers: [transformerDirectives(), transformerVariantGroup(), transformerCompileClass()],
    }),
    sveltekit()
  ]
};

export default config;
 ```
 
 ```svelte
<div>
  <p text-red-200>Index Page</p>
</div>
 ```
 
![Screenshot_20230114_112445](https://user-images.githubusercontent.com/96418027/212458144-e20985bd-a411-4018-94e3-29e82532f009.png)

**Comment by shedali** [comment]
what's the difference between importing attributify from unocss and the documented `import presetAttributify from '@unocss/preset-attributify'`

**Comment by impcyber** [comment]
@aadi58002 Maybe you need to use 
https://github.com/unocss/unocss/tree/main/packages/preset-attributify#valueless-attributify

[transformer-attributify-jsx](https://github.com/unocss/unocss/tree/main/packages/transformer-attributify-jsx)

**Comment by sibbng** [comment]
Attributify support for svelte-scoped mode added in #2260 and released in [v0.50.2](https://github.com/unocss/unocss/releases/tag/v0.50.2)
